test/force-auto.py.in

```diff
@@ -303,6 +299,7 @@ class T_Arch_Latest(Test):
    config = "arch"
    base = "archlinux:latest"
    arch_excludes = ["aarch64", "ppc64le"]  # base only amd64
+   force_excludes = ["seccomp"]  # issue #FIXME
```

Comment:
Did you want to open this issue and fix this `FIXME` comment prior to getting this PR merged in?